.github/ISSUE_TEMPLATE/functional_bug.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Functional Bug
+about: Use this template to report functionally incorrect or misleading documentation.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Summary
+
+Replace this with a short summary (one sentence or short paragraph) of
+the issue you are describing.
+
+## Affected paths
+
+List the URL(s) where you found faulty behavior. You can use URLs from
+[the rendered site](https://docs.cleura.cloud) or from [the sources on
+GitHub](https://github.com/citynetwork/docs), both is fine.
```

Comment:
Replace "both is fine" (grammatically incorrect) with "either is fine".

Reply:
Will do.